src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzer.cs

```diff
@@ -490,10 +490,20 @@ private static void ReportDiagnostics(IOperation operation, SmallDictionary<stri
                 IInvocationOperation iOperation => iOperation.TargetMethod,
                 IObjectCreationOperation cOperation => cOperation.Constructor,
                 IFieldReferenceOperation fOperation => IsWithinConditionalOperation(fOperation) ? null : fOperation.Field,
+                IPropertyReferenceOperation pOperation => GetPropertyAccessor(pOperation.Property, operation),
                 IMemberReferenceOperation mOperation => mOperation.Member,
                 _ => null,
             };
 
+        private static ISymbol GetPropertyAccessor(IPropertySymbol property, IOperation operation)
+            => operation.GetValueUsageInfo(property.ContainingSymbol) switch
```

Comment:
I believe the `ValueUsageInfo.ReadWrite` should be handled to return both the getter and setter, which would mean `ISymbol? GetOperationSymbol` should be changed to `IEnumerable<ISymbol>? GetOperationSymbols`?

Reply:
Well, the calls site and the entire code path is not expecting multiple symbols. and i thought `ValueUsageInfo.ReadWrite` will be covered by separate Read/Write as `ReadWrite = Read | Write` but now i think it might not, will check

Reply:
Yeah, I think `WindowsOnlyPropertySetter += value;` will not be flagged with current implementation, when it should be.